app/src/main/java/org/oppia/app/activity/HtmlParserTestActivty.kt

```diff
@@ -1,22 +1,26 @@
 package org.oppia.app.activity
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.text.Spannable
 import android.widget.TextView
 import org.oppia.app.R
 import org.oppia.util.parser.HtmlParser
+import org.oppia.util.parser.UrlImageParser
+import javax.inject.Inject
 
 /** This is a dummy activity to test Html parsing. */
-class HtmlParserTestActivty : AppCompatActivity() {
+class HtmlParserTestActivty : InjectableAppCompatActivity() {
+  @Inject
+  lateinit var urlImageParser: UrlImageParser
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
+    activityComponent.inject(this)
     setContentView(R.layout.activity_test_html_parser)
 
     var testHtmlContentTextView = findViewById(R.id.test_html_content_text_view) as TextView
```

Comment:
Prefer writing it this way

`val testHtmlContentTextView: TextView = findViewById(R.id.test_html_content_text_view)`


I suggest try to solve warnings suggested by Android Studio.

Reply:
Okay will do that.